applications/network_radar_pipeline/cpp/metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+	"application": {
+		"name": "Networked Radar Pipeline",
+		"authors": [
+			{
+				"name": "Dylan Eustice",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"language": "C++",
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk_version": "0.6.0",
+		"platforms": ["amd64", "arm64"],
+		"tags": ["Networking", "Network", "UDP", "IP", "Signal Processing", "RADAR"],
+		"ranking": 1,
```

Comment:
@dylan-eustice I checked with @jjomier, we don't have cicd setup to run/test this application so for now we let's update the ranking to 2 until we have cicd testing setup for this.